src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java

```diff
@@ -31,32 +32,64 @@
  * </ul>
  *
  */
-public final class RMSMappingQuality extends InfoFieldAnnotation implements StandardAnnotation {
+public final class RMSMappingQuality extends InfoFieldAnnotation implements StandardAnnotation, ReducibleAnnotation {
```

Comment:
in GATK4 I deleted the `RMSAnnotation` abstract class and the `AS_RMSAnnotation` abstract class because each had exactly one subclass and the generalization feels premature